dev_docs/key_concepts/api_authorization.mdx

```diff
@@ -0,0 +1,309 @@
+---
+id: kibDevDocsSecurityAPIAuthorization
+slug: /kibana-dev-docs/key-concepts/security-api-authorization
+title: Kibana API authorization
+description: This guide provides an overview of API authorization in Kibana.
+date: 2024-10-04
+tags: ['kibana', 'dev', 'contributor', 'security']
+---
+
+Authorization is an important aspect of API design. It must be considered for all endpoints, even those marked as `internal` . This guide explains how and when to apply authorization to your endpoints
+
+This guide specifically focuses on API authorization:
```

Comment:
> just a list of the most important
I wanted to highlight the most important, but if it's confusing I can make it as table of contents

> I'm wondering if we can just use normal case and use capital letters only where necessary, but it's minor.
I'm happy to convert it

Reply:
> I wanted to highlight the most important, but if it's confusing I can make it as table of contents

Got it, no strong preference - it might just be me. Whatever you feel is best, would work for me too.

> I'm happy to convert it

That'd be great, thank you!